enable/tests/test_assistant_test_case.py

```diff
@@ -3,8 +3,9 @@
 except ImportError:
     import mock
 import nose
+from unittest import skipIf
 
-from traitsui.tests._tools import skip_if_null
+from traitsui.tests._tools import is_null
```

Comment:
Currently we have pinned CI to TraitsUI 7.0.1 when https://github.com/enthought/enable/issues/425 has yet to be fixed. Importing private module should be avoided as much as possible, especially for something this trivial, it is not worth the dependency. Could you please use `unittest.skipIf(ETSConfig.toolkit == "null")`?

Reply:
Or revert back to the original for now as this is orthogonal?

Reply:
done

Reply:
Thank you! Sorry for the inconvenience there.